public/index.html

```diff
@@ -16,6 +16,21 @@ <h1>Carebnb</h1>
       from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> 
       is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a>
       <a href="http://chittagongit.com/icon/share-icon-svg-5.html">Share Icon Svg #31948</a>
```

Comment:
http://chittagongit.com/icon/share-icon-svg-5.html seems to be a jpg not a svg. But either works!

Reply:
Yeah, luckily this one was pretty much spot on for what the icon looks like on Airbnb. I will consider going after fixing this when I cut a new branch to fix all of my svg's that are currently img tags.